packages/react-mutation-mapper/src/model/MutationMapperStore.ts

```diff
@@ -23,7 +23,7 @@ export interface MutationMapperStore {
     gene: Gene;
     dataStore: DataStore;
     uniprotId: RemoteData<string | undefined>;
-    activeTranscript?: string;
```

Comment:
@onursumer why `activeTranscript` is optional here?

Reply:
I think this should have been defined as `activeTranscript: string | undefined;` 

Maybe we update this to make sure that it is never undefined. But there might be cases where the gene is invalid or for some reason no transcript can be found for a given gene.